### DISCUSSION TITLE
What is the "Menu attached" functionality

### DISCUSSION THREAD

**Discussion by leepowellnbs** [VirtusLab/strapi-plugin-navigation#194] [General]
Excuse the question, but I'm new to this plugin and trying to understand what the "Menu attached" functionality does, or what it is intended to do? I couldn't see anything in the docs.

**Comment by cyp3rius** [comment]
Hello @leepowellnbs,

It's an indication if you would like to see this item in the menu or just be part of navigation structure for your client app routing. Based on various use cases we've decided to expose it for everyone as community seen the value :) You might use it / handle by your client app but you don't need to.

**Reply by konhi** [reply]
Maybe it would be a good idea to add that information to README and description below the field? This thing feels quite vague!

**Comment by leepowellnbs** [comment]
Ah i see - thanks for the explanation :)
